starter-code/src/components/ProductTable.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from "react";
+import ProductRow from "./ProductRow.js";
+
+class ProductTable extends Component {
+  render() {
+    return (
+      <div>
+        <table>
+          {this.props.products.data
+            .filter(product => product.name.includes(this.props.searchedText))
+            .map(product => {
+              if (this.props.filterCheck) {
+                if (product.stocked) {
+                  return <ProductRow myProduct={product} />;
```

Comment:
Tu aurais pu passer en props toutes les propriétés du `product` en faisant `<ProductRow {...product} />`. Comme ça dans `ProductRow`tu pourras faire `this.props.price|stocked|name` au lieu de `this.props.myProduct.price|stocked|name`